jenkins/bootstrap_test.py

```diff
@@ -2199,7 +2203,8 @@ def test_valid_job_envs(self):
                     if (
                             '--env-file=jobs/platform/gce.env' not in args
                             and '--env-file=jobs/platform/gke.env' not in args
-                            and '--check-leaked-resources=true' in args):
+                            and '--check-leaked-resources=true' in args
+                            and 'generated' not in config[job].get('tags', [])):
```

Comment:
Is there a reason why we need this? I would expect this to follow the same logic, regardless of whether or not the job is generated?

Maybe the problem here is that this was generated before we checked in this change?

AKA I recommend we just remove the `'generated' not in` part.